recipe/0001-fix-makefile.patch

```diff
@@ -0,0 +1,35 @@
+diff -ruN Raven-rev322/Makefile Raven-rev322-patch/Makefile
+--- Raven-rev322/Makefile	2021-04-23 18:48:03.000000000 +0200
++++ Raven-rev322-patch/Makefile	2021-05-12 09:51:59.504952715 +0200
+@@ -12,17 +12,17 @@
+ #            Jan 2019    Alan J. Barton - added OS name to created exe name
+ ###
+ # appname := raven_rev$(shell svnversion -n . | sed -e 's/:/-/g')_$(shell uname -o).exe
+-appname := Raven.exe
++appname := raven
+ 
+-CXX      := g++
+-CXXFLAGS := -Wno-deprecated
++override CXXFLAGS := $(CXXFLAGS) -Wno-deprecated
+ 
+ srcfiles := $(shell find . -name "*.cpp")
+ objects  := $(patsubst %.cpp, %.o, $(srcfiles))
+ 
+ # include netcdf
+-CXXFLAGS += -Dnetcdf                # if netcdf is installed use "CXXFLAGS += -Dnetcdf",                    else "CXXFLAGS += "
+-LDLIBS   := -L/usr/local -lnetcdf   # if netcdf is installed give first path "-L<PATH>"and then "-lnetcdf", else "LDLIBS   := "
++CXXFLAGS += -Dnetcdf
++CXXFLAGS += $(shell nc-config --cflags)
```

Comment:
This could probably be a single line but I wanted to be explicit.  The second line is the important bit that adds the include path you need under OSX (and probably Linux, too -- it may be working there for the wrong reasons).